quiz.html

```diff
@@ -3,53 +3,121 @@
 	<head>
 		<title>Skin type Quiz</title>
 		<link rel="stylesheet" href="style.css"/>
+        <script type="text/javascript" src="quiz.js"></script>
 	</head>
 
 	<body>
 		<header>
-			<p>This will be the nav</p>
+			<a href="index.html" class="logo">Skinclusive</a>
+			<nav name="top">
+				<ul>
+					<li><a href="quiz.html">skin type quiz</a></li>
+					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin products</a>
+							<a href="#">dry skin products</a>
+							<a href="#">combo skin products</a>
+							<a href="#">sensitive skin products</a>
+						</div>
+					</li>
+					<li class="dropdown"><a href="article-list.html">articles &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin articles</a>
+							<a href="#">dry skin articles</a>
+							<a href="#">combo skin articles</a>
+							<a href="#">sensitive skin articles</a>
+						</div>
+					</li>
+				</ul>
+			</nav>
 		</header>
 		
 		<div class="wrapper">
 			<div class="quiz-page">
 				<h1>Skin Type Quiz</h1>
 
-				<!-- This div is for the skin type quiz and is different to the other pages -->
-				<div class="quizbox">
-					<h1>Question 1 heading</h1>
-					<h2>1 / 5</h2>
-					<p>Question related images and text will go here</p>
-					<hr color="white" width="80%" size="3">
-
-					<h1>Question 2 heading</h1>
-					<h2>2 / 5</h2>
-					<p>Question related images and text will go here</p>
-					<hr color="white" width="80%" size="3">
-
-					<h1>Question 3 heading</h1>
-					<h2>3 / 5</h2>
-					<p>Question related images and text will go here</p>
-					<hr color="white" width="80%" size="3">
-
-					<h1>Question 4 heading</h1>
-					<h2>4 / 5</h2>
-					<p>Question related images and text will go here</p>
-					<hr color="white" width="80%" size="3">
+				<!-- This form is for the skin type quiz and is different to the other pages -->

```

Comment:
In my final layout, I changed the HTML where necessary so that the quiz would actually be functional when linked to the javascript file.